public class App {
    public static void main(String[] args) throws Exception {
<<<<<<< Updated upstream
        LeeZhuPasswords pass = new LeeZhuPasswords("Abc123&!!!!!!!!!!!");
        System.out.println(pass.determineStrength());
//        pass = new LeeZhuPasswords("A!1ba^!!");
//        System.out.println(pass.determineStrength());
=======
        
//         LeeZhuPasswords pass = new LeeZhuPasswords("ABcDe123!");
// //        System.out.println(pass.determineStrength());
//         pass = new LeeZhuPasswords("ABcDe123!");
//         System.out.println(pass.determineStrength());
>>>>>>> Stashed changes
//        pass = new LeeZhuPasswords("Password12#@");
//        System.out.println(pass.determineStrength());
//        pass = new LeeZhuPasswords("StrongPass123@#%$");
//        System.out.println(pass.determineStrength());
//        pass = new LeeZhuPasswords("NoDigits!");
//        System.out.println(pass.determineStrength());
<<<<<<< Updated upstream
//        new PasswordApp(new PasswordPannel());
=======
        LeeZhuPasswords pass5 = new LeeZhuPasswords("zyx78");//to test 5
        System.out.println(pass5.consecutive());
>>>>>>> Stashed changes
    }
}
